README.ci.md

```diff
@@ -1,14 +1,14 @@
 **WARNING:** This document is a work in progress and intended as a RFC.
-If you are not a Coq Developer, don't follow this instructions yet.
+If you are not a Coq Developer, don't follow these instructions yet.
 
 Introduction
 ============
 
-The Coq Travis CI infrastructure is meant to provide lightweight
-automatics testing of pull requests.
-If you are on GitLab, their integrated CI is also set up.
+As of 2017, lightweight tests are automatically run against all pull requests
+or merge requests submitted to Coq's official GitHub or GitLab repositories,
```

Comment:
The rewording is incorrect but might indicate that the initial text wasn't clear. There is no official GitLab repo at least for now. The GitLab CI settings are just meant for developers who wish to test code before submitting a PR, similarly to what GitHub users can do with a Travis account thanks to the Travis settings.